avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastDeserializerGeneratorForReuseTest.java

```diff
@@ -207,11 +207,13 @@ public void testFastGenericDeserializerPrimitFloatList() throws Exception {
     // new record array length shorter than reuse record.
     arrayList.clear();
     arrayList.add((float)10);
+    arrayList.add((float)20);
+    arrayList.add((float)30);
```

Comment:
This change seems to contradict the comment...? Shall we change the comment or the code?

Reply:
Sure. I will update the comment